cypress/elements/fileMenu/save.js

```diff
@@ -1,24 +1,25 @@
 import {
     clickFileMenuButton,
-    FILE_MENU_BUTTON_SAVE,
+    FILE_MENU_BUTTON_SAVE_EXISTING,
+    FILE_MENU_BUTTON_SAVE_NEW,
     FILE_MENU_BUTTON_SAVEAS,
 } from '.'
 import { clickMenuBarFileButton } from '../menuBar'
 
-const saveModalEl = '*[class^="MuiDialog-container"]' // TODO: Add data-test to save modal to target this better
-const saveModalSaveButtonEl = '[type="submit"]'
+const saveModalEl = '[data-test="file-menu-saveas-modal"]'
+const saveModalSaveButtonEl = '[data-test="file-menu-saveas-modal-save"]'
 
 export const saveNewAO = (name, description) => {
     clickMenuBarFileButton()
-    clickFileMenuButton(FILE_MENU_BUTTON_SAVE)
+    clickFileMenuButton(FILE_MENU_BUTTON_SAVE_NEW)
     cy.get(saveModalEl).find('input').clear().type(name)
```

Comment:
Change to get the `file-menu-saveas-modal-name` instead.
If you don't want to use `getBySel(X)` directly only you can chain it like `cy.get(saveModalEl).findBySel(X)`.
Same goes for the textarea on the next line.

Reply:
Done.